cmd/ddev/cmd/global_dotddev_assets/commands/web/blackfire

```diff
@@ -44,14 +48,13 @@ case $1 in
   status)
     php --version | grep "with blackfire" >/dev/null 2>&1
     phpstatus=$?
-    killall -0 blackfire-agent 2>/dev/null
-    agentstatus=$?
+    # Can't use killall here because it kills this process!
+    agentstatus=$(ps -ef | awk '$8~/^blackfire.*/ { print $2 }' 2>/dev/null)
+    if [ "${pid}" != "" ]; then kill $pid; fi
```

Comment:
Why killing the process here? `killall` was used in the first place, passing `-0` to check if it is running, right? AFAICS, the same thing is done thanks to `ps -ef`.
IMHO, the kill command should be removed. Otherwise checking the status would also stop the agent, which seems to be counter-productive :thinking:

Reply:
`killall` kills this process, which is named "blackfire", so that's why I had to change it.

Pushed another commit to remove the copy-pasta error that kills the pid in `ddev blackfire status`. Good eye, thanks!

Reply:
Cool thanks 👍

Reply:
Thanks @rfay and @lolautruche. It seems we are good to go! I love when a plan comes together ;)

Reply:
Sure glad you fixed this up. I hadn't even become aware of the new blackfire version and deprecation.